packages/jest-parallel/README.md

```diff
@@ -0,0 +1,173 @@
+# jest-parallel
+
+Module for executing heavy tasks under forked processes in parallel, by providing a modern interface (`Promise` based), minimum overhead, and sticky workers.
+
+The module works by providing an absolute path of the module to be loaded in all forked processes, as well as the list of methods that can be remotely called. All methods are exposed on the parent process as promises, so they can be `await`'ed. Child (worker) methods can either be synchronous or asynchronous.
+
+The way sticky workers work is by using the returned string of the `getWorkerKey` method. If the string was used before, the call will be queued to the related worker; if not, it will be executed by the first available worker, then sticked to the worker that executed it; so the next time it will be processed by the same worker. If you have no preference on the worker executing the task, but you have defined a `getWorkerKey` method because you want _some_ of the tasks to be sticked, you can return `null` from it.
```

Comment:
Can you explain at first what a sticky worker is meant to be before you explain how to use it? Instead of saying "sticked", can you say "bound"? "it will be executed by the first available worker, then bound to the that same worker for subsequent calls"